### ISSUE TITLE
Casing of labels, titles, buttons, etc.

### ISSUE THREAD

**Issue by jeremycook** [OrchardCMS/OrchardCore#1553]
As an admin or site visitor I would like to see consistent use of casing within individual pages, across pages, and across modules. I would like to see a formal recommendation for the English language. I'm focusing on English because casing can vary between languages, and this one recommendation may be unreasonable for other languages. My thinking is that by default other languages would follow the formal English casing rule, but users of and translators to another language could choose to provide a formal recommendation when it varies from English. This topic could be expanded into a discussion of [content style](https://www.drupal.org/drupalorg/style-guide/content) but for this issue and the moment I am only interested in casing.

An example of mixed casing: field labels on the Setup page are sometimes "Title Cased," and sometimes "Sentence cased."

![image](https://user-images.githubusercontent.com/452414/36915697-e2a0f43a-1e0e-11e8-955e-27334c067ad2.png)

Inconsistency is most commonly seen in:

* Form labels
* Form buttons
* Headings
* Menu items
* Taxonomies
* Titles, in general
* ?

**Comment by sebastienros** [comment]
GTG, any way you think is better. I assume "Sentence case" is more common, and works everywhere.

**Comment by jeremycook** [comment]
I agree that "Sentence case" is more common. I find it's easier to get right. I'd recommend it for all non-sentences including button text and page titles.

Here's a definition from Wiktionary.org that covers the basics:

> The standard capitalisation of an English sentence, with the first letter uppercase and subsequent letter lowercase with exceptions such as [proper nouns](https://en.wiktionary.org/wiki/proper_noun) or [acronyms](https://en.wiktionary.org/wiki/acronym). https://en.wiktionary.org/wiki/sentence_case

**Comment by jeremycook** [comment]
Is there interest in adding that definition or something like it somewhere in docs? It applies to so many areas (localization, module development, theme development, menu management, ...) that I'm not sure where it should go.

**Comment by Piedone** [comment]
Our new contrib docs could be updated with this: https://docs.orchardcore.net/en/latest/docs/guides/contributing/.
